cmd/kubeadm/app/phases/certs/certs_test.go

```diff
@@ -43,7 +43,7 @@ func TestCreatePKIAssets(t *testing.T) {
 			expected: false,
 		},
 		{
-			// CIDR too small
+			// IPv4 CIDR too short
```

Comment:
The comments could be used in a new "name" field, to identify the test and make it more self documenting.